src/client/editor/SceneLoader.js

```diff
@@ -73,12 +81,25 @@ function shallowEquals(objA, objB) {
   return true;
 }
 
-function isEmptyObject(obj) {
-  for (const key in obj) {
-    return false;
+function hasExtrasOrExtensions(obj) {
+  for (const key in obj.userData) {
+    if (!key.startsWith("_")) {
```

Comment:
Check for has `hasOwnProperty` here, otherwise this will return true for all objects because of keys like `'__proto__'`.